configs/experimental/windows/0.1.0/bazel_a320e4460f35767da7fc8eb824ce1f4b304b5e0f/BUILD.bazel

```diff
@@ -0,0 +1,148 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
what does the "a320e4460f35767da7fc8eb824ce1f4b304b5e0f" correspond to?
we'd rather this be bazel_<version> or something else humanly readable

Reply:
Changed to bazel_experimental. Will figure out versioning story when we get to it.